### PR TITLE
added logic for admin adding product

### PR DESCRIPTION
**Pull request by ashwoodaa**

### CODE REVIEW

File: bangazon.py [hunk 1]
```diff
@@ -116,8 +116,20 @@ def select_payment_type(self): # will move to within Order Process
...
   def create_product_type(self):
     self.page_clear()
-    print('Welcome Admin')
+    print('Welcome, Admin, to the **Add A Product** Page')
     time.sleep(.5)
+
+    print("\nLet's add a product")
+    add_product = input('Enter Product Name: ')
+    add_product_price = input('Enter Product Price: ')
+    new_product = New_Product(add_product, add_product_price)
```
Comment (keithbradley1) [inline bangazon.py]: new_product can be 'product'

**Comment by keithbradley1** [comment]
girl your code is so hot its melting my brain right now

**Comment by SimonHerrera** [comment]
:facepunch:

**Comment by ashwoodaa** [comment]
@keithbradley1 Are you Tennessee? 'Cause you're hot as hell

**Pull request merged**